packages/playground/_stories/main/FileUploader/FileUploader.stories.ts

```diff
@@ -97,4 +90,11 @@ Advanced.parameters = {
 			inline: false,
 		},
 	}
+};
+
+export const Custom = Template.bind({});
```

Comment:
Think we'd better remove this sample all together. We could potentially think for a meaningful sample with a custom design in the future.